### ISSUE TITLE
[BULK]: Textures for unfolded_block particles and door being mined. Deleting rifts with blocks and eternal fluid. Messages cut off by screen bounds. Door failure after sending mobs.

### ISSUE THREAD

**Issue by XenoTheStrange** [DimensionalDevelopment/DimDoors#233]
Breaking dimdoors:unfolded_block results in black/pink 'missing texture'
![2021-09-05_13 12 20](https://user-images.githubusercontent.com/48196283/132140223-f6daad6e-2cfb-4598-bcab-9c2d22d64fb6.png)

**Comment by XenoTheStrange** [comment]
I fixed it for me personally by modifying this file in the .jar:
dimdoors/models/block/unfolded_block.json

Original Contents:
{
  "parent": "block/cube",
  "textures": {
    "down": "dimdoors:block/unfolded_block_down",
    "up": "dimdoors:block/unfolded_block_up",
    "north": "dimdoors:block/unfolded_block_north",
    "south": "dimdoors:block/unfolded_block_south",
    "west": "dimdoors:block/unfolded_block_west",
    "east": "dimdoors:block/unfolded_block_east"
  }
}


Modified:
{
  "parent": "block/cube",
  "textures": {
    "particle": "dimdoors:block/unfolded_block",
    "down": "dimdoors:block/unfolded_block_down",
    "up": "dimdoors:block/unfolded_block_up",
    "north": "dimdoors:block/unfolded_block_north",
    "south": "dimdoors:block/unfolded_block_south",
    "west": "dimdoors:block/unfolded_block_west",
    "east": "dimdoors:block/unfolded_block_east"
  }
}

Changes:
(Line 4) "particle": "dimdoors:block/unfolded_block",

Result:
![2021-09-05_13 47 47](https://user-images.githubusercontent.com/48196283/132141017-7991cf1e-c7d4-4fdf-910d-14399807c8b1.png)

I do not know how mods are structured before being packed, and I don't generally github, so I don't know how to create a pull request.

**Comment by XenoTheStrange** [comment]
Mod message doesn't consider in game bounds. Ex. placing a door in a pocket with an unstable rift.
I don't know what it says.
![image](https://user-images.githubusercontent.com/48196283/132144980-6c3fef86-7c7f-49b4-bdda-f4eed97e90cd.png)

**Comment by XenoTheStrange** [comment]
Mining active doors gives off a weird texture:
![image](https://user-images.githubusercontent.com/48196283/132145547-7c4890af-6f6f-4cd0-9345-980b1eeff98a.png)

**Comment by XenoTheStrange** [comment]
Rifts can be deleted by replacing the rift block with any other. This might be intended but it sort of invalidates the "Close Rift" item. I feel like only doors should be able to go there (dimdoors:gold_door, minecraft:oak_door (etc, wooden doors), and so on.)

Eternal fluid can't be interacted with like a normal liquid. No reason to really, but you can't place blocks through it or replace it with anything else.
Messing with blocks around the liquid may cause the lot of it to start disappearing (???)
![image](https://user-images.githubusercontent.com/48196283/132145849-7bb7faa8-bb59-4bd3-87bb-37f351c26794.png)

**Comment by XenoTheStrange** [comment]
Wandering through limbo I came across a door portal that didn't have a door with it. Destroyed it on accident by placing a block.

**Comment by XenoTheStrange** [comment]
After sending some creatures and villagers through a quartz door to my personal pocket, that door stopped working in the world. "failed to teleport you, please report bug"
Villagers did disappear through the overworld quartz door, but did not appear within the pocket. They did teleport successfully with the iron door though.

EDIT: after restarting minecraft (or reloading the world?) I could enter my personal pocket again and the villagers were present inside.

**Comment by XenoTheStrange** [comment]
In the config If you have "doors close behind you" enabled, they also close behind all entities that travel through, including thrown items. That could be an option, but this should only apply to players I think.

**Comment by XenoTheStrange** [comment]
Gold door behavior -- If you place a gold door within your personal pocket, it may result in the creation of a new area (intended)
or
it will warp you some distance outside of the cube in your personal pocket, such that you can still see it in the distance as you're dying to the void. Is this intended? Maybe a sort of door to limbo or for players with Elytras?? This makes some gold doors only send you to limbo, and without a way of closing the portal that's doing it (because it's in the void outside of an indestructible cube.
????

**Comment by XenoTheStrange** [comment]
Using a rift signature I created portals linking my base and personal pocket.
I went into my pocket using the normal portal and tried to exit using the primary quartz door.
The last exterior quartz door I entered through no longer existed and I had nowhere to go.

This message appeared (Roughly):
The rift you used to enter the pocket dimension has closed around you and you have entered limbo!"

The door closed behind me, but I didn't go anywhere.  I re-opened the door and walked back out.
